src/components/CallView/Video.vue

```diff
@@ -231,6 +241,10 @@ export default {
 			}
 		},
 
+		forceMute() {
+			this.model.forceMute()
```

Comment:
This should only work for moderators

Reply:
It only works for moderators now (except when muting guests due to a bug unrelated to this pull request which I am working on).

However, if only moderators can mute other participants then no one in a file conversation can mute anyone. And even in our instance nobody can mute anyone else during company calls or other calls in main rooms in which there are no moderators.

Due to this I would rather go with what @jospoortvliet proposed in #282 and allow anyone logged in to mute others.

Reply:
> Due to this I would rather go with what @jospoortvliet proposed in #282 and allow anyone logged in to mute others.

Normal users being able to mute others is not really somethign you would want to have. Especially since there is no visual indicator who did that. Just imagine a troll in the team muting the speaking person all the time.

> However, if only moderators can mute other participants then no one in a file conversation can mute anyone. And even in our instance nobody can mute anyone else during company calls or other calls in main rooms in which there are no moderators.

This is an acceptable flaw until https://github.com/nextcloud/spreed/issues/2236 is being handled

Reply:
> Especially since there is no visual indicator who did that.

I added a message to be shown to the participant being muted. Currently it does not specify who muted you, but it should be doable if needed (that is, if muting was finally changed to be available for any logged in user; if muting is kept only for moderators then I do not think that it is so important to know who muted you).

Reply:
Muting others is a feature mostly focusing on webinaries. So yes, it must be limited to moderators.